### PR TITLE
Add Publish content to Crafter CLI develop -Ticket 843

### PR DESCRIPTION
**Pull request by Euquimides**
Added Publish Content Option to Crafter CLI, following ticket:
https://github.com/craftersoftware/craftercms/issues/843

### CODE REVIEW

File: src/main/groovy/org/craftercms/cli/commands/site/PublishContent.groovy [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            throw new CommandLine.ParameterException(commandSpec.commandLine(), 'Invalid publishing target. Use live or staging')
+        }
+    }
+
+    def run(client) {
+        // Tokenize items and optionalDependencies
+        def items = this.items.tokenize(',')
+        def optionalDependencies = this.optionalDependencies ? this.optionalDependencies.tokenize(',') : null
+
+        def path = '/studio/api/2/workflow/publish.json'
```
Comment (Euquimides) [inline src/main/groovy/org/craftercms/cli/commands/site/PublishContent.groovy:62]: Notice that this endpoint will be deprecated in the future based on: https://github.com/craftercms/studio/pull/3544/files#diff-855ee91a131d9de0f962d9f7c1631e86abf9712d9347bcdeb32b6a265f2bfbbdR6624

I will change this once the new APIs are implemented.

**Review by yacdaniel — APPROVED** [review]

File: src/main/groovy/org/craftercms/cli/commands/site/PublishContent.groovy [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        def result = client.post(path, query)
+        if (!result) {
+            return
+        }
+        if (schedule) {
+            println(result.response.message)
+            println "The selected content has been scheduled to be published to ${publishingTarget} at ${schedule}"
+        } else {
+            println(result.response.message)
+            println "The selected content has been published to ${publishingTarget}"
```
Comment (sumerjabri) [inline src/main/groovy/org/craftercms/cli/commands/site/PublishContent.groovy:80]: submitted to be published

File: README.md [hunk 3]
```diff
@@ -26,6 +26,26 @@ crafter-cli create-site -e local -s my-site -n My Site \
 
 `crafter-cli sync-from -e local -s editorial -n origin`
 
+Also, you are able to publish an item to live or staging (staging must be enabled first before using that target):
```
Comment (sumerjabri) [inline README.md:29]: You can publish content to the Live or Staging targets (Staging must be enabled to publish to it):

File: README.md [hunk 4]
```diff
@@ -26,6 +26,26 @@ crafter-cli create-site -e local -s my-site -n My Site \
...
+
+`crafter-cli publish-content -e local -s editorial --publishingTarget live --items "/site/website/index.xml"`
+
+`crafter-cli publish-content -e local -s editorial --publishingTarget staging --items "/site/website/index.xml"`
+
+To publish various items to live or staging, you can use the following command (be wary of the quotes and spaces):
```
Comment (sumerjabri) [inline README.md:35]: To publish multiple items, separate them by comma.

**Review by sumerjabri — CHANGES_REQUESTED** [review]
See my notes

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**